services/user-tenant-service/README.md

```diff
@@ -0,0 +1,135 @@
+ ### Setting Environment Variables ( Application Properties)
+
+Do not forget to set Environment variables in .env. The examples below show a common configuration for a PostgreSQL Database running locally. (in `src/main/resources/application.properties`).
+
+```properties
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url=jdbc:postgresql://localhost:5432/userTenant
+spring.datasource.username=user
+spring.datasource.password=pass
+
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+# Hibernate ddl auto (create, create-drop, validate, update)
+spring.jpa.hibernate.ddl-auto=
+
+spring.jpa.properties.hibernate.auto_quote_keyword=true
+spring.jpa.show-sql=true
+```
+| Name          | Required | Default Value | Description                                                                                                                        |
+| ------------- |----------|---------------| ---------------------------------------------------------------------------------------------------------------------------------- |
+| spring.datasource.username | Y        |               | Login username of the database. |
+| spring.datasource.password | Y        |               | Login password of the database. |
+| spring.datasource.url | Y        |               | JDBC URL of the database. |
+| spring.jpa.show-sql | N        | false         | Whether to enable logging of SQL statements. |
+| spring.jpa.hibernate.ddl-auto | N        |               | DDL mode |
+| spring.jpa.properties.hibernat.dialect | Y        |               | Dialect in Hibernate class. |
+| spring.liquibase.enabled | N        | true          | Whether to enable Liquibase support. |
+|spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation | Y        | false         |Is used in the Spring Framework's Hibernate integration to control the behavior of Large Object (LOB) handling|
+|spring.jpa.properties.hibernate.auto_quote_keyword| Y        | false         |The property spring.jpa.properties.hibernate.auto_quote_keyword is used to control the automatic quoting of SQL keywords in Hibernate queries.|
+### API Documentation
+
+#### Common Headers
+
+Authorization: Bearer <token> where <token> is a JWT token signed using JWT issuer and secret.
+`Content-Type: application/json` in the response and in request if the API method is NOT GET
+
+#### Common Request path Parameters
+
+{version}: Defines the API Version
+
+### Common Responses
+
+200: Successful Response. Response body varies w.r.t API
+401: Unauthorized: The JWT token is missing or invalid
+403: Forbidden : Not allowed to execute the concerned API
+404: Entity Not Found
+400: Bad Request (Error message varies w.r.t API)
+201: No content: Empty Response
+
+## API's Details
+
+Visit the [OpenAPI spec docs](./openapi.md)
```

Comment:
We can remove this section since it points to a non-existing page. We'll add this again when we figure out a way to generate a markdown with openapi spec.